generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics-garbagecollector/metrics-garbagecollector.component.html.ejs

```diff
@@ -65,7 +65,7 @@
         </div>
     </div>
 
-    <div class="col-md-4">
+    <div id="garbageCollectorMetrics" class="col-md-4">
```

Comment:
This is unrelated to table from where `aria-describedby` refers. Maybe move title `metrics.jvm.gc.title` from `metrics.component.html` to this component and refer to that header?

Reply:
done